src/Core/Form/IdentifiableObject/DataHandler/EmployeeFormDataHandler.php

```diff
@@ -160,6 +171,12 @@ public function update($id, array $data)
             ->setHasEnabledGravatar((bool) $data['has_enabled_gravatar'])
         ;
 
+        /** @var UploadedFile $uploadedAvatar */
+        $uploadedAvatar = $data['avatarUrl'];
+        if ($uploadedAvatar instanceof UploadedFile) {
+            $this->imageUploader->upload($id, $uploadedAvatar);
+        }
+
```

Comment:
I think you could even move this even lower after the `$this->bus->handle($command);` no?
To make sure the command has been executed correctly before trying to handle the image upload.

Unless some validation midway prevents you from doing it?

Reply:
Yep I was focused on my problem and didn't think about the rest of the function...
Thanks for advice ^^